viewer/src/datamodels/cad/CadModelBudget.ts

```diff
@@ -22,8 +24,16 @@ export type CadModelSectorBudget = {
   readonly maximumNumberOfDrawCalls: number;
 };
 
-export const defaultCadModelSectorBudget = {
-  highDetailProximityThreshold: 10,
-  geometryDownloadSizeBytes: 35 * 1024 * 1024,
-  maximumNumberOfDrawCalls: 1000
-};
+export const defaultCadModelSectorBudget = isMobileOrTablet()
```

Comment:
```suggestion
export const defaultCadModelSectorBudget: CadModelBudget = isMobileOrTablet()
```

Here should be a type. CadModelBudget or CadModelSectorBudget not really sure.